pkgs/development/python-modules/pymisp/default.nix

```
@@ -0,0 +1,32 @@
+{ lib, buildPythonPackage, fetchPypi, python3, pytest
+, requests, python-dateutil, jsonschema, deprecated
+}:
+
+buildPythonPackage rec {
+  version = "2.4.174";
+  pname = "pymisp";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "173247f368e78c9069d201621f3d7746b41cc478d76b923e1a4c1e7f29ada287";
+  };
```

```suggestion
  src = fetchFromGitHub {
    owner = "MISP";
    repo = "PyMISP";
    rev = "v${version}";
    hash = "sha256-q67LDJiwibG1X9QtzYQSP/qSTSXDSHp60gJIa7coq+A=";
  };
```

This makes sure the tests are run (they are removed when published)